src/main/resources/assets/mmorpg/lang/ru_ru.json

```
@@ -383,981 +211,204 @@
-	"mmorpg.support_gem.magic_shield_on_hit": "Поддержка восстановления магического щита при ударе",
-	"mmorpg.support_gem.mana_on_hit": "Поддержка восстановления маны при ударе",
-	"mmorpg.support_gem.mana_saver_dmg": "Поддержка экономии маны при нанесении урона",
-	"mmorpg.support_gem.melee_damage": "Поддержка урона в ближнем бою",
-	"mmorpg.support_gem.physical_damage": "Поддержка физического урона",
-	"mmorpg.support_gem.physical_pene": "Поддержка пробивания физического сопротивления",
-	"mmorpg.support_gem.plus_aoe": "Поддержка увеличенной области",
-	"mmorpg.support_gem.poison_chance": "Поддержка шанса отравления",
-	"mmorpg.support_gem.proc_freeze": "Поддержка ледокола",
-	"mmorpg.support_gem.proc_shock": "Shock Support Gem",
-	"mmorpg.support_gem.proj_speed": "Поддержка ускорения снарядов",
-	"mmorpg.support_gem.ranged_damage": "Поддержка урона в дальнем бою",
-	"mmorpg.support_gem.rare_cooldown": "Поддержка большего времени восстановления редких умений",
-	"mmorpg.support_gem.rare_mana_eff": "Поддержка увеличения затрат маны редких умений",
-	"mmorpg.support_gem.small_cdr": "Поддержка уменьшения времени восстановления малого умения",
-	"mmorpg.support_gem.spell_damage": "Поддержка урона от умений",
-	"mmorpg.support_gem.spell_damage_no_crit": "Поддержка ограниченного разрушения",
-	"mmorpg.support_gem.summon_damage": "Поддержка урона от призванных существ",
-	"mmorpg.support_gem.summon_duration": "Поддержка времени существования призванных существ",
-	"mmorpg.support_gem.summon_health": "Поддержка здоровья призванных существ",
-	"mmorpg.support_gem.totem_damage": "Поддержка урона от тотемов",
-	"mmorpg.support_gem.totem_dmg_cdr": "Поддержка увеличенного времени восстановления урона от тотемов",
-	"mmorpg.support_gem.water_pene": "Поддержка пробивания холодом",
-
-"_comment": " [CATEGORY]: SPELLS - NAMES",
-
-
-"_comment": " [CATEGORY]: RARITIES - NAMES",
+"_comment": " [CATEGORY]: CHAT MESSAGES - NAMES",
 
-	"mmorpg.rarity..common": "Обычный",
-	"mmorpg.rarity..epic": "Епический",
-	"mmorpg.rarity..legendary": "Легендарный",
-	"mmorpg.rarity..mythic": "Мистический",
-	"mmorpg.rarity..rare": "Редкий",
-	"mmorpg.rarity..runeword": "Слово руны",
-	"mmorpg.rarity..uncommon": "Необычный",
-	"mmorpg.rarity..unique": "Уникальный",
+	"mmorpg.chat.already_casting": "Уже производится заклинание",
+	"mmorpg.chat.alt_to_show_other_spell": "[Alt], чтобы показать другое заклинание",
+	"mmorpg.chat.any_item": "Любое оружие может быть использовано.",
+	"mmorpg.chat.cant_equip_that_many_supports": "Вы не можете экипировать столько поддерживающих самоцветов! Вы можете увеличить количество слотов, повысив уровень навыка.",
+	"mmorpg.chat.cant_have_more_gems_than_runes_in_runeword": "Предметы с руными словами не могут иметь больше самоцветов, чем рун, вставленных в них",
+	"mmorpg.chat.cant_use_multiple_same_supports": "Нельзя использовать несколько одинаковых поддерживающих самоцветов.",
+	"mmorpg.chat.cast_failed": "[Не удалось произвести заклинание]: ",
```

Поменяй, пожалуйста, везде где сможешь найти
```suggestion
	"mmorpg.chat.cast_failed": "[Не удалось произнести заклинание]: ",
```

---

исправлено